src/test/java/br/com/sunty/dao/CategoryDaoTest.java

```diff
@@ -0,0 +1,66 @@
+package br.com.sunty.dao;
+
+import br.com.sunty.models.category.Category;
+import br.com.sunty.models.category.CategoryBuilder;
+import br.com.sunty.utils.JPAUtil;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.persistence.EntityManager;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CategoryDaoTest {
+
+    private EntityManager entityManager;
+    private CategoryDao categoryDao;
+
+    @BeforeEach
+    void setUp() {
+        entityManager = JPAUtil.getEntityManager();
+        categoryDao = new CategoryDao(entityManager);
+        entityManager.getTransaction().begin();
+    }
+
+    @AfterEach
+    void tearDown() {
+        entityManager.getTransaction().rollback();
+    }
+
+    @Test
+    void shouldValidateOrderOfActiveCategories() {
```

Comment:
Somente sugestão:

```suggestion
    void findAllActiveOrdered__should_return_active_categories_ordered_by_orderToShow() {
```
Na Alura seguimos uma padrão para criar os nomes dos métodos de testes, usamos o padrão `snake_case` e no começo dos testes adicionamos o nome do método que estamos testando